### PR TITLE
Adding Shimmer 

### PR DESCRIPTION
**Pull request by aman7870**
https://user-images.githubusercontent.com/26053172/105190646-3b016c80-5b5c-11eb-89ee-2968240a5d49.mp4

adding the shimmer feature. I have used setTimeout() for delaying the data for 3sec.

### CODE REVIEW

File: src/pages/openPRs.tsx [hunk 1]
```diff
@@ -14,11 +15,18 @@ type pullRequestType = {
...
   useEffect(() => {
     (async () => {
+      setLoading(true);
+
       const response = await fetch({ url });
       setPullRequests(response.data.pullRequests);
+
+      setTimeout(() => {
+        setLoading(false);
+      }, 3000);
```
Comment (whyDontI) [inline src/pages/openPRs.tsx:29]: setTimeout is not required, we can remove this.

File: src/pages/openPRs.tsx [hunk 2]
```diff
@@ -27,14 +35,22 @@ const openPRs: FC = () => {
...
+              key={link}
+              title={title}
+              username={username}
+              createdAt={createdAt}
+              updatedAt={updatedAt}
+              url={link}
+            />
+          )
+        }
+      </>
```
Comment (whyDontI) [inline src/pages/openPRs.tsx:53]: https://www.loom.com/share/20569612ef244c58a6599e6c7bc170cf

**Review by whyDontI — CHANGES_REQUESTED** [review]
Nice work @aman7870 🥳 
Requested some little changes.

**Comment by aman7870** [comment]
did the changes and removed the function getLoading(). Please Review

**Review by whyDontI — APPROVED** [review]
Great work @aman7870 🥳

**Comment by aman7870** [comment]
Thanks for reviewing and merging the PR

**Pull request merged**